config/initializers/app_routes.rb

```diff
@@ -2,6 +2,6 @@ module AppRoutes
   # memoize this due to a bug in MRI that shows up in multi-threaded code
   # see https://github.com/puma/puma/issues/647, search for "bug in MRI"
   def self.url_helpers
-    @@routes_url_helpers ||= Rails.application.routes.url_helpers
+    @routes_url_helpers ||= Rails.application.routes.url_helpers
```

Comment:
it is a class method, so `@` should be fine / a global cache, why would we want to delete the file ?